std/algorithm/comparison.d

```diff
@@ -689,6 +698,54 @@ if (isInputRange!R1 && isInputRange!R2)
     }
 }
 
+/// ditto
+int cmp(alias pred, R1, R2)(R1 r1, R2 r2)
+if (isInputRange!R1 && isInputRange!R2)
+{
+    static if (is(typeof(pred) : string)
+        && (__traits(compiles, { static assert("a < b" == pred); })))
+    {
+        // In case someone was explicitly using the default predicate.
```

Comment:
please remove, no need to accommodate this case

Reply:
Removed